sql/sql_base.cc

```diff
@@ -8997,7 +8997,7 @@ fill_record(THD *thd, TABLE *table_arg, List<Item> &fields, List<Item> &values,
   List_iterator_fast<Item> f(fields),v(values);
   Item *value, *fld;
   Item_field *field;
-  Field *rfield;
+  Field *rfield= nullptr;
```

Comment:
Is this related to this patch?

Reply:
Sorry, no, it was included accidentally. This fix should actually be applied to the 10.5 branch to avoid `-Wmaybe-uninitialized` on `g++ -Og`.